samconfig.toml

```diff
@@ -5,7 +5,7 @@ version = 0.1
 [default]
 [default.global.parameters]
 stack_name = "beobots"
-region = "eu-central-1"
+region = "us-east-1"
```

Comment:
Is there a reason to choose us-east-1 by default? Cause currently we aiming to deploy in southeast europe.